src/js/burials/validation.js

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
+
+export function validateBurialDate(errors, page) {
```

Comment:
I think we would write separate validation functions for each one, though it might make sense to write a generic function for it (we have a date range validator, but it assumes particular date field names).

That the date is in the past should be checked by the validation included in the `currentOrPastDateUI` definition.